AGENTS.md

```diff
@@ -0,0 +1,59 @@
+# Cockpit starter-kit - Technical Overview
+
+This document provides a technical overview for applications or plugins that integrate into Cockpit and are prepared with the starter-kit. Is is human readable but intended to give AI context to follow the project standards.
+
+## Project Structure
+
+- **Upstream Website**: [Cockpit Project](https://cockpit-project.org/)
```

Comment:
This is somewhat true, but not specific for the forked project. It's also not useful for AI agents.